server/webapp/WEB-INF/rails/webpack/views/components/tooltip/index.scss

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@import "../../global/common";
+@import "../icons/index";
```

Comment:
This import will cause duplicate css to be generated (as you can see in the correspoinding ts file).

If all we want is an icon to be shown on the tooltip, it's probably better to just add it here.

Reply:
@ketan - I agree with you on the css duplication. But I didn't want to break the convention of defining icons at one place. 
Though, at the same time, I don't see why we need to use `info-circle` icon or `question-circle` icon independently.

For this specific case, we can move icons to the tooltip section (to avoid messing with imports or extending classes). If we need to use these 2 icons somewhere else, then we need to define them again in icons.